my-sms/app/controllers/courses_controller.rb

```diff
@@ -94,4 +94,8 @@ def load_course
   def flash_errors
     super(@course)
   end
+
+  def flash_errors
+    super(@course)
+  end
```

Comment:
repeated method?

Reply:
Don't know how that snuck in there. Removed that commit.